honeybee_energy/properties.py

```diff
@@ -0,0 +1,46 @@
+"""Energy Properties."""
+from .boundarycondition import BoundaryConditions
+
+
+class EnergyProperties(object):
```

Comment:
Am I correct in understanding that this gets assigned to the Honeybee Face as a property?

Reply:
Yes. That is correct.